service/population.go

```diff
@@ -20,29 +21,43 @@ func NewPopulationService(populationRepository repository.PopulationRepository)
 	}
 }
 
-func (p *populationService) GetPopulationStatistics() ([]model.PopulationResponseItem, error) {
-	out := []model.PopulationResponseItem{}
+func (p *populationService) errorMessage(err error, functionName string) {
+	logrus.WithFields(logrus.Fields{
+		"Module":  "Service",
+		"Funtion": functionName,
+	}).Error(err)
+}
+
+func (p *populationService) queryLog(message string, functionName string) {
+	logrus.WithFields(logrus.Fields{
+		"Module":  "Service",
+		"Funtion": functionName,
+	}).Info(message)
+}
```

Comment:
Merge this into
```suggestion
func (p *populationService) generateLogger(functionName string) *logrus.Entry {
	return logrus.WithFields(logrus.Fields{
		"Module":  "Service",
		"Funtion": functionName,
	})
}
```

Reply:
@sirateek should we provide the err as well? (same like the handler)

Reply:
fixed